packages/itwinui-react/src/core/Table/index.ts

```diff
@@ -5,6 +5,8 @@
 export { Table } from './Table.js';
 export type { TableProps, TablePaginatorRendererProps } from './Table.js';
 
```

Comment:
nit: remove extra line.

Reply:
I added a blank line on L7 since L8's imports comes from another file, and not `./Table.js` like L5 and L6.